tests/end2end/features/tabs.feature

```diff
@@ -1010,6 +1010,54 @@ Feature: Tab management
         And I run :buffer "1/2/3"
         Then the error "No matching tab for: 1/2/3" should be shown
 
+    # :tab-take
```

Comment:
Seems like those fail, at least on AppVeyor: https://ci.appveyor.com/project/qutebrowser/qutebrowser/build/master-2047/job/ni5b9kjv1ygpkeqg

I haven't looked into it yet though. It's quite possible you can't tes tthis easily without having a window manager to consistently focus windows. See the corresponding tests in https://github.com/qutebrowser/qutebrowser/pull/2783/files